14-microbiome-diversity.Rmd

```diff
@@ -51,7 +51,14 @@ by species abundances.
 **Dominance** indices are in general negatively correlated with
   diversity, and sometimes used in ecological literature. High
   dominance is obtained when one or few species have a high share of
-  the total species abundance in the community.
+  the total species abundance in the community.  
+  
+**Phylogenetic diversity** was first proposed by [@Faith1992], unlike the 
+  diversity measures mentioned above, Phylogenetic diversity (PD) 
+  measure incorporates information from phylogenetic relationships 
+  stored in `phylo` tree between species in a community (sample). The 
+  Faith's PD is calculated as the sum of branch length of all species in 
+  a community (sample).
```

Comment:
We do not have Faith included in mia::estimateAlphaDiversity at the moment? Should we?

Reply:
Another option would be to make PR to picante, adding `TSE` support.

Reply:
If we can improve the speed from `picante` then it would be useful to add. For only one function adding a dependency/suggests is arguable.

Reply:
Agree. But TSE support in picante itself wouldn't hurt.